src/test/java/io/jenkins/pluginhealth/scoring/probes/UpdateCenterPluginPublicationProbeTest.java

```diff
@@ -69,4 +69,26 @@ public void shouldFailIfPluginIsNotInUpdateCenterMap() {
             .comparingOnlyFields("id", "status", "message")
             .isEqualTo(ProbeResult.failure(UpdateCenterPluginPublicationProbe.KEY, "This plugin does not exists in update-center"));
     }
+
+    @Test
+    public void shouldSucceedIfPluginIsInUpdateCenterMap() {
+        final Plugin plugin = mock(Plugin.class);
+        final ProbeContext ctx = mock(ProbeContext.class);
+        final String pluginName = "foo";
+
+        when(plugin.getName()).thenReturn(pluginName);
```

Comment:
Hey @alecharp 

I was adding the test case for the following:

> You are missing the test case when everything is fine.

If I am not wrong I should populate the following

```
 when(ctx.getUpdateCenter()).thenReturn(new UpdateCenter(
            Map.of(),
            Map.of(),
            Collections.emptyList()
        ));
```

for the test case to succeed. 

But I am was not able to figure out what values to add in ` Map.of()`, ` Map.of()`, `Collections.emptyList()`

Reply:
Let's go on this together so we can be corrected if we go wrong somewhere: what do you mean by this test case "when everything is fine"?

Reply:
"When everything is fine" is based on one of previous reviews: https://github.com/jenkins-infra/plugin-health-scoring/pull/205#discussion_r1102858205

Reply:
Since I did not write any tests cases that don't fail. I assumed that's what it meant.

Reply:
Yeah, I was more interested in trying to understand what it means for the test case when everything is fine. As per my understanding, it means that the UpdateCenter `If the plugin can be located in the map, then the probe should return a SUCCESS as the plugin is still published.`
Taken from this initial comment by Adrien: https://github.com/jenkins-infra/plugin-health-scoring/issues/178#issuecomment-1413456490

Does this help in understanding what you should be filling in the `new UpdateCenter()`?